LayoutTests/fast/css/view-transitions-nested-transparency-layers.html

```diff
@@ -0,0 +1,33 @@
+<html class="reftest-wait">
+<script>
+    function finishTest() {
+        document.documentElement.classList.remove("reftest-wait");
+    }
+    async function startTest() {
+        document.startViewTransition(() => requestAnimationFrame(() => requestAnimationFrame(finishTest)));
+    }
+    onload = () => requestAnimationFrame(() => requestAnimationFrame(startTest));
+</script>
+<style>
+    .parent {
+        width: 100px;
+        height: 100px;
+        background: red;
+        opacity: 0.5;
+    }
+    .target {
+        width: 100px;
+        height: 100px;
+        contain: paint;
+        background: blue;
+        opacity: 0.5;
+        view-transition-name: target;
+    }
+    html::view-transition-group(target) { animation-duration: 300s; }
+    html::view-transition-new(target) { animation: unset; opacity: 0.5; }
+    html::view-transition-old(target) { animation: unset; opacity: 0.5; }
```

Comment:
```suggestion
    html::view-transition-new(target) { animation: unset; opacity: 0; }
    html::view-transition-old(target) { animation: unset; opacity: 1; }
```

This forces the page to always display the old snapshot (instead of crossfading between the new and old), which is the one that was broken.